rts/motoko-rts/src/types.rs

```diff
@@ -367,7 +425,7 @@ pub const TAG_CLOSURE: Tag = 11;
 pub const TAG_SOME: Tag = 13;
 pub const TAG_VARIANT: Tag = 15;
 pub const TAG_BLOB: Tag = 17;
-pub const TAG_FWD_PTR: Tag = 19;
+pub const TAG_FWD_PTR: Tag = 19; // Only used by the copying GC - not to be confused with forwarding pointer in the header used for incremental GC.
```

Comment:
> "Only used by the copying GC"

A more precise statement would be "Only used by non-incremental GCs" ? As a future cleanup, would it make sense to use the new forwarding fields in the old GCs as well?

Reply:
Currently, this special tag/record is only used by the copying GC (not by the other GCs). Maybe we should in future even revise which GCs should stay in the runtime system. We could indeed use the forwarding pointer in the copying GC - not sure whether I should already unify it now. I will keep it in my backlog.